apiserver/upgradeservice/upgradeimpl.go

```diff
@@ -91,17 +131,24 @@ func CreateUpgrade(request *msgs.CreateUpgradeRequest, ns string) msgs.CreateUpg
 			Spec: spec,
 		}
 
-		// remove any existing pgtask for this minor upgrade
-		result := crv1.Pgtask{}
-		found, err := kubeapi.Getpgtask(apiserver.RESTClient,
-			&result, spec.Name, ns)
-		if found {
-			err := kubeapi.Deletepgtask(apiserver.RESTClient, spec.Name, ns)
-			if err != nil {
-				response.Status.Code = msgs.Error
-				response.Status.Msg = err.Error()
-				return response
+		// remove any existing pgtask for this upgrade
+		task := crv1.Pgtask{}
+		found, err := kubeapi.Getpgtask(apiserver.RESTClient, &task, spec.Name, ns)
+
+		/*
+			if found {
+				err := kubeapi.Deletepgtask(apiserver.RESTClient, spec.Name, ns)
+				if err != nil {
+					response.Status.Code = msgs.Error
+					response.Status.Msg = err.Error()
+					return response
+				}
 			}
+		*/
```

Comment:
Assuming this can be removed.

Reply:
for sure. Another thing I **thought** I removed, but it's gone now.